recipes/mobile-broadband-provider-info/mobile-broadband-provider-info.inc

```diff
@@ -0,0 +1,6 @@
+DESCRIPTION = "mobile broadband settings for different service providers in different countries"
+LICENSE = "CC-PD"
+
+SRC_URI = "https://git.gnome.org/browse/mobile-broadband-provider-info/snapshot/${PN}-${PV}.tar.xz"
+
+inherit autotools-autoreconf
```

Comment:
Yep, the tarball doesn't come with a configure script, so we need to run autoreconf.